receiver/kafkametricsreceiver/scraper_test_helper.go

```diff
@@ -153,14 +159,50 @@ func (s *mockClusterAdmin) ListConsumerGroupOffsets(string, map[string][]int32)
 	return s.consumerGroupOffsets, nil
 }
 
+func (s *mockClusterAdmin) DescribeConfig(cr sarama.ConfigResource) ([]sarama.ConfigEntry, error) {
+	topicName := cr.Name
+	if cr.Type == sarama.BrokerResource {
+		return s.brokerConfigs, nil
+	}
+	if s.topics[topicName].ConfigEntries == nil {
+		return nil, fmt.Errorf("no config entries found for topic")
+	}
+	configEntry := make([]sarama.ConfigEntry, 1)
+	for name, entry := range s.topics[topicName].ConfigEntries {
+		configEntry = append(configEntry, sarama.ConfigEntry{
+			Name:  name,
+			Value: *entry,
+		})
+	}
+	return configEntry, nil
+}
+
 func newMockClusterAdmin() *mockClusterAdmin {
 	clusterAdmin := new(mockClusterAdmin)
 	r := make(map[string]string)
 	r[testGroup] = testGroup
 	clusterAdmin.consumerGroups = r
 
+	strMinInsyncReplicas := strconv.Itoa(testMinInsyncReplicas)
+	strLogRetentionMs := strconv.Itoa(testLogRetentionMs)
+	strLogRetentionBytes := strconv.Itoa(testLogRetentionBytes)
+
+	brokerConfigEntry := sarama.ConfigEntry{
+		Name:  "log.retention.hours",
+		Value: strconv.Itoa(testLogRetentionHours),
+	}
+	configEntries := make([]sarama.ConfigEntry, 1)
+	configEntries = append(configEntries, brokerConfigEntry)
+	clusterAdmin.brokerConfigs = configEntries
 	td := make(map[string]sarama.TopicDetail)
-	td[testTopic] = sarama.TopicDetail{}
+	td[testTopic] = sarama.TopicDetail{
+		ReplicationFactor: testReplicationFactor,
+		ConfigEntries: map[string]*string{
+			"min.insync.replicas": &strMinInsyncReplicas,
```

Comment:
We have a few kafka metric strings that are used multiple times that could instead be central variables, such as `min.insync.replicas`, `retention.ms`, `retention.bytes`, and `log.retention.hours`.